spec/rails_helper.rb

```diff
@@ -15,6 +15,7 @@
 Dir[Rails.root.join("spec", "support", "**", "*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
+  config.fixture_path = File.expand_path("../fixtures/", __FILE__)
   # If we want Capybara + DatabaseCleaner + Poltergeist to work correctly, we
   # have to just set this to false.
   config.fixture_path = File.expand_path("../fixtures/", __FILE__)
```

Comment:
I guess you should remove this one and keep the other line since the comment doesn't make sense for this.